javascript/gdx_getting_started_1.html

```diff
@@ -0,0 +1,61 @@
+  <!--
+  This example uses the GDX JavaScript library to collect all of the sensor information from a Vernier GoDirect Sensor. 
+  Press the 'Select GoDirect Device' button to open communication to the device in a Chrome browser. 
+  This example does not collect data from a sensor. 
+  
+  This program uses the following functions from the GDX Library:
+  
+  gdxDevice.selectDevice();
+  gdxDevice.close();
+  gdxDevice.sensors.filter();
+  -->  
+
+
+<!doctype html>
+<html>
+<head>
+  <title>Example 1: Showing Sensor Information</title>
+  <meta name="description" content="The gdx_getting_started_1.html example shows how you can use the gdx functions to list information on all of the sensor channels on a Go Direct device. This includes sensor channel number, sensor description, units, and 'incompatible sensors'">
```

Comment:
Looks like the "incompatible sensors" portion was removed from this example, so this comment is not accurate.

Reply:
I add it.